roles/ceph-facts/tasks/facts.yml

```diff
@@ -290,6 +290,18 @@
       ntp_service_name: ntpd
     when: ansible_os_family in ['RedHat', 'Suse']
 
+- name: set chrony daemon name RedHat and Ubuntu based OSs
+  block:
```

Comment:
I don't think we need a block for this

Reply:
I like tying similar things into a block.

Reply:
agree, but I know rishabh loves `block` :laughing:

Reply:
:P